typings/internal-ambient.d.ts

```diff
@@ -16,6 +16,7 @@ declare namespace NodeJS {
   interface V8UtilBinding {
     getHiddenValue<T>(obj: any, key: string): T;
     setHiddenValue<T>(obj: any, key: string, value: T): void;
+    deleteHiddenValue<T>(obj: any, key: string): T;
```

Comment:
`deleteHiddenValue` returns `void` --> https://github.com/electron/electron/blob/master/atom/common/api/atom_api_v8_util.cc#L82

```suggestion
    deleteHiddenValue(obj: any, key: string): void;
```